static/app/views/performance/landing/widgets/components/widgetContainer.tsx

```diff
@@ -244,15 +243,18 @@ export const WidgetInteractiveTitle = ({
       options={menuOptions}
       value={chartSetting}
       onChange={handleChange}
-      renderWrapAs={TextOverflow}
       triggerProps={{borderless: true, size: 'zero'}}
     />
   );
 };
 
 const StyledCompactSelect = styled(CompactSelect)`
+  min-width: 0;
+  font-weight: 400;
+  margin: -${space(0.5)} -${space(0.75)} 0;
```

Comment:
Negative margins to offset the space added by button's padding and maintain vertical text alignment